adapter.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+	"sync"
+
+	handler "github.com/yogeshprasad/fargate/backend"
+	"github.com/yogeshprasad/fargate/storage"
+)
+
+func main() {
+	// All parameters are optional
+	storageDriver := flag.String("storage_driver", "", "Storage driver to send data to")
+	storageDriverOptStr := flag.String("storage_driver_options", "", `Storage driver options e.g "key1=value1, key2=value2"`)
+	port := flag.Int("port", 0, "Port to listen")
+
+	flag.Parse()
+
+	// Waitgroup to keep track of goroutines
+	var wg sync.WaitGroup
+
+	// Map to support the list of storage driver
+	funcs := map[string]func(map[string]string, *sync.WaitGroup){"wavefront": storage.Wavefront}
+
+	// Do not use the storage driver if user is not intended
+	if *storageDriver == "" {
+		log.Printf("Storage driver is not supplied")
```

Comment:
Should we exit here?

Reply:
as of now, it is optional, so a user can debug the collected metrics without supplying the storage driver.

Reply:
Can we just add a `--test` or `--debug` flag or such for that. So we'll always quit if driver is not supplied. But if debug flag is provided we will only log the metrics etc.